packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,183 @@
+  focusVisible?: boolean;
+  hover?: boolean;
+  children?: any;
+}
+
+const arabicDecorator = (Story) => (
+  <div dir="rtl" lang="ar">
+    {Story()}
+  </div>
+);
+
+const TextareaStory = ({ children, focus, focusVisible, hover, ...args }: TextareaStoryProps) => {
+  const classNames = {
+    'utrecht-textarea--focus': focus,
+    'utrecht-textarea--focus-visible': focusVisible,
+    'utrecht-textarea--hover': hover,
+  };
+
+  return <Textarea className={clsx(classNames)} defaultValue={children} {...args} />;
+};
+
+const meta = {
+  title: 'CSS Component/Textarea',
+  id: 'css-textarea',
+  component: TextareaStory,
+  argTypes: {
+    disabled: {
+      description: 'Disabled',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'Invalid',
+      control: 'boolean',
+    },
+    readOnly: {
+      description: 'Read-only',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Required',
+      control: 'boolean',
+    },
+    focus: {
+      description: 'Focus',
+      control: 'boolean',
+    },
+    focusVisible: {
+      description: 'focusVisible',
+      control: 'boolean',
+    },
+    spellCheck: {
+      description: 'Spellcheck',
+      control: { type: 'select' },
+      options: ['', 'false', 'true'],
+    },
+  },
+  args: {
+    dir: 'auto',
+    disabled: false,
+    focus: false,
+    focusVisible: false,
+    hover: false,
+    invalid: false,
+    readOnly: false,
+    required: false,
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  tags: ['autodocs'],
+  parameters: {
+    status: {
+      type: 'WORK IN PROGRESS',
+    },
+    tokensPrefix: 'utrecht-textarea',
+    tokens,
+    tokensDefinition,
+    docs: {
+      description: {
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof Textarea>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: `Styling via the \`.utrecht-textarea\` class name.`,
+      },
+    },
+  },
+};
+
+export const PlaceholderRightToLeft: Story = {
+  name: 'Placeholder (right-to-left)',
+  args: {
+    ...Default.args,
+    placeholder: 'اكتب رسالتك',
+    children: '',
+  },
+  decorators: [arabicDecorator],
+};
+
+export const RightToLeft: Story = {
+  args: {
+    ...Default.args,
+    children:
+      'لكن لا بد أن أوضح لك أن كل هذه الأفكار المغلوطة حول استنكار  النشوة وتمجيد الألم نشأت بالفعل، وسأعرض لك التفاصيل لتكتشف حقيقة وأساس تلك السعادة البشرية، فلا أحد يرفض أو يكره أو يتجنب الشعور بالسعادة، ولكن بفضل هؤلاء الأشخاص الذين لا يدركون بأن السعادة لا بد أن نستشعرها بصورة أكثر عقلانية ومنطقية فيعرضهم هذا لمواجهة الظروف الأليمة، وأكرر بأنه لا يوجد من يرغب في الحب ونيل المنال ويتلذذ بالآلام، الألم هو الألم ولكن نتيجة لظروف ما قد تكمن السعاده فيما نتحمله من كد وأسي.',
+  },
+  decorators: [arabicDecorator],
+};
```

Comment:
Storybook gebruikt de naam van de export (`RightToLeft`) als naam van de story, als er geen `name` is in gesteld. Voor namen die met leestekens worden geschreven, kun je de `name` property gebruiken. De story name was eerst `"Right-to-left"`, nu is 'ie `"Right To Left"`. In dit soort gevallen is het nodig om expliciet de `name` in te stellen.